app/controllers/criteria_controller.rb

```diff
@@ -160,7 +160,9 @@ def upload
             type = criterion_yml[1]['type']
             begin
               if type.casecmp('rubric') == 0
-                criterion = RubricCriterion.load_from_yml(criterion_yml)
+                data = RubricCriterion.load_from_yml(criterion_yml)
```

Comment:
Use unpacking assignment here: `criterion, levels = ...`.

Reply:
@ashih2018 I actually meant directly on line 163 (so no need for the `data` variable).